desc/compute/_equil.py

```diff
@@ -73,23 +115,106 @@ def _J_sup_zeta(params, transforms, profiles, data, **kwargs):
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["J^rho", "J^theta", "J^zeta", "e_rho", "e_theta", "e_zeta"],
+    data=[
+        "J^rho",
+        "J^zeta",
+        "J^theta*sqrt(g)",
+        "e_rho",
+        "e_zeta",
+        "e_theta/sqrt(g)",
+    ],
 )
 def _J(params, transforms, profiles, data, **kwargs):
     data["J"] = (
         data["J^rho"] * data["e_rho"].T
-        + data["J^theta"] * data["e_theta"].T
+        + data["J^theta*sqrt(g)"] * data["e_theta/sqrt(g)"].T
         + data["J^zeta"] * data["e_zeta"].T
     ).T
     return data
 
 
+@register_compute_fun(
+    name="J*sqrt(g)",
+    label="\\mathbf{J} \\sqrt{g}",
+    units="A m",
+    units_long="Ampere meters",
+    description="Plasma current density weighted by 3-D volume Jacobian",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=[
+        "B_rho_z",
+        "B_theta_r",
+        "B_zeta_t",
+        "B_rho_t",
+        "B_theta_z",
+        "B_zeta_r",
+        "e_rho",
+        "e_theta",
+        "e_zeta",
+    ],
+)
+def _J_sqrt_g(params, transforms, profiles, data, **kwargs):
+    data["J*sqrt(g)"] = (
+        (data["B_zeta_t"] - data["B_theta_z"]) * data["e_rho"].T
+        + (data["B_rho_z"] - data["B_zeta_r"]) * data["e_theta"].T
+        + (data["B_theta_r"] - data["B_rho_t"]) * data["e_zeta"].T
+    ).T / mu_0
+    return data
+
+
+@register_compute_fun(
+    name="(J*sqrt(g))_r",
+    label="\\partial_{\\rho} (\\mathbf{J} \\sqrt{g})",
+    units="A m",
+    units_long="Ampere meters",
+    description="Plasma current density weighted by 3-D volume Jacobian",
```

Comment:
is this the radial derivative?

Reply:
I'll fix this and the comment in `plotting.py`